.github/workflows/multinode.yml

```diff
@@ -75,6 +79,8 @@ on:
         required: true
       OS_APPLICATION_CREDENTIAL_SECRET:
         required: true
+      SLACK_WEBHOOK_URL:
+        required: true
```

Comment:
Required but enable_slack_alert is optional?

Reply:
I guess it's not strictly required.